api/turing/models/ensembling_job.go

```diff
@@ -12,12 +12,12 @@ import (
 // EnsemblingJob holds the information required for an ensembling job to be done asynchronously
 type EnsemblingJob struct {
 	Model
-	Name            string       `json:"name"`
-	EnsemblerID     ID           `json:"ensembler_id"`
+	Name            string       `json:"name" validate:"required"`
+	EnsemblerID     ID           `json:"ensembler_id" validate:"required"`
 	ProjectID       ID           `json:"project_id"`
 	EnvironmentName string       `json:"environment_name"`
```

Comment:
If the environment is simply going to be taken from the Ensembler, does it need to be stored here?

Reply:
Not quite related to this PR but I do agree with you. I think it was there last time because I thought it was a user supplied value, but during the review process this was changed to a configuration value and this was left behind.

Reply:
Ah seems like I missed out an important bit of information. There might be a chance next time where this option might be fillable by the user. It would be easier to state the default environment for now in the database as compared to doing a data patch in the future (which might be complicated because the default environment is pulled from MLP).